controllers/comments.controller.js

```diff
@@ -0,0 +1,15 @@
+const { selectCommentsByArtId } = require('../models/comments.model')
+
+exports.getCommentsByArtId = ( req, res, next) => {
+    const articleId = req.params.article_id
+    selectCommentsByArtId(articleId)
+        .then((comments) => {
+            comments.length === 0 ?
```

Comment:
we should check this in our model where our model returns a rejected promise so that a catch block here can pass this on to the error handling middleware